lerna.json

```diff
@@ -1,5 +1,5 @@
 {
-  "lerna": "3.18.4",
+  "lerna": "3.20.2",
```

Comment:
This field is legacy and can be removed now: https://github.com/lerna/lerna#legacy-fields

Reply:
oh, removed in january 2018. heh